apps/www/lib/redirects.js

```diff
@@ -2814,6 +2814,11 @@ module.exports = [
     source: '/docs/guides/resources/examples',
     destination: '/docs/guides/getting-started',
   },
+  {
+    permanent: true,
+    source: '/docs/guides/auth/auth-email-templates',
```

Comment:
This page actually exists though: https://supabase.com/docs/guides/auth/auth-email-templates

You sure you want a redirect and not updating the link?

Reply:
Oh wait, my bad, I'm using the wrong source 🤦🏼‍♀️